client/extensions/woocommerce/lib/countries/CA.js

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
See this discussion thread: https://github.com/Automattic/wp-calypso/pull/15321/files/26a24132eb6fc7d65089391e0df37d71b0245345#r123366902

We will have to do something hardcoded for the initial setup since WooCommerce might not be installed on the remote site yet. After v1 it would be nice to not have to hard code every single thing. Maybe we could expose them via a site we control and pull from there.